xdp.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29728.190
+# Visual Studio Version 17
```

Comment:
@mtfriesen - is the version change on sln file acceptable?

Reply:
The VS2019 build still works in CI, so this should be fine. We tend to edit these by hand FWIW.